DESCRIPTION

```diff
@@ -1,18 +1,21 @@
-Package: ihme.covid
-Title: Shared functions for IHME's covid-19 research
-Version: 0.0.1
-Authors@R:
-    person(given = "Mike",
-           family = "Richards",
-           role = c("aut", "cre"),
-           email = "miker985@uw.edu",
-           comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: Shared functions for IHME COVID-19 work.
-Encoding: UTF-8
-Imports:
-    data.table,
-    rgdal,
-    rprojroot
-LazyData: true
-License: BSD
-RoxygenNote: 7.0.2
+Package: ihme.covid
+Title: Shared functions for IHME's covid-19 research
+Version: 0.0.1
+Authors@R:
+    person(given = "Mike",
+           family = "Richards",
+           role = c("aut", "cre"),
+           email = "miker985@uw.edu",
+           comment = c(ORCID = "YOUR-ORCID-ID"))

```

Comment:
Consider adding yourself as an author:

* wrap the authors in `c()`
* add yourself
* make sure last line has no trailing `,`

[example here](https://stash.ihme.washington.edu/users/miker985/repos/lbd.mbg/browse/DESCRIPTION#7-21)